push-notification-ios.js

```diff
@@ -0,0 +1,34 @@
+import * as Ably from 'ably';
```

Comment:
I'm not familiar with the react native ecosystem - is it the case that the vast majority of react native installations are of a version that supports es6 modules?

Reply:
React Native supports these features out of the box